Etiquette/CODE_REVIEW.md

```diff
@@ -59,7 +59,7 @@ Reviewing a Pull Request
 * Do not try to make everything perfect (it never will be and otherwise we'll never deliver anything), but do not compromise on quality
 * Do not comment if you don't have any suggestions to address your comment
 * Use GitHub suggestions as much as possible
-* If you feel strong against the change - request changes and explain why you are doing that. Do not request changes for typos, missing documentation comments or code style violations, trust your colleagues to address them.
+* If you feel strong against the change - please use `Request changes` and explain why you are doing that. **Any comment left in the PR is taken as a suggestion, and it's up to the author's discretion to accept them or not, except if you use `Request changes`.**
```

Comment:
If this implies that such comments can be simply ignored without any response on them then I don't think that this is a good etiquette.

If the comment is something reviewer does not have a strong opinion on and does not care about what author of the change thinks about such suggestion ("take it or leave it") then they should indeed leave it up to the author of the change to accept it, but then they should just approve the change (assuming there is nothing else). 

If reviewer does not have a strong opinion but at the same time wants to discuss the matter before deciding to approve or request changes than just leaving a comment is reasonable imo. It starts conversation. It's fine if reviewer is not ready to approve or request changes and needs clarification or some discussion before that.

Maybe what we are missing is this conclusion part. So such discussion should maybe result in the comment being resolved (with "resolve" button) and approval if there is nothing left or with request changes if the result of conversation is that changes should be made. (though I thought that's clear for everyone)

We also have a point about such discussions where it seems to hard to reach conclusion quickly to better happen not on GitHub but in person or in a call.

Reply:
When reviewer just goes and requests changes for something that author of the change considers as a matter of taste this will as well cause a conflict. That's why I think people in this case tend to leave a comment and neither approve, nor request changes

Reply:
I agree with what @ilyapuchka is suggesting. Not every comment has to be addressed, but ideally every comment has to be (at least!) acknowledged.

Reply:
This doesn't imply what you mentioned @ilya. Please @diego add an addendum to this with your final sentence, since it's a clean and to the point rule.

Reply:
Added your suggestion Diego. Thank you.